app/models/gsa18f/procurement.rb

```diff
@@ -10,6 +10,13 @@ class Procurement < ActiveRecord::Base
     URGENCY = DATA['URGENCY']
     OFFICES = DATA['OFFICES']
     RECURRENCE = DATA['RECURRENCE']
+    PURCHASE_TYPES = {
+      "Software" => 0,
+      "Training/Event" => 1,
+      "Office Supply/Miscellaneous" => 2
+    }
```

Comment:
You might add a comment to the effect of "careful changing these numbers, as they will still be present in the database".